Tasks/PublishSymbols/task.json

```diff
@@ -34,7 +34,7 @@
         {
             "name": "SearchPattern",
             "type": "string",
-            "label": "Search Pattern",
+            "label": "Search pattern",
```

Comment:
I made all of the casing consistent for the labels on this task. Now they all follow are consistent with "Continue on error" which I believe is the more appropriate casing convention for labels anyway.